docs/build/blast-install-1.md

```diff
@@ -39,31 +29,13 @@ Checkout [Setup Rust](/docs/build/setup-rust)
 
 ## 02 Install Cudos Blast
 
-### Globally - Create a new project anywhere
```

Comment:
Why is this being removed?

Global install is the recommended way to install Cudos Blast.

There is really no reason to install a project-local Cudos Blast, other than possibly that you don't have permission to install it globally.

[There is a better solution that allows installing it just for your user](https://github.com/sindresorhus/guides/blob/main/npm-global-without-sudo.md), but if you don't have permission to install it globally, you most likely also don't have permission to start up a local Cudos node for testing.

Reply:
Done